File: how-to-sqrt_or_more.cpp
#include <iostream>
#include <string>
#include <cmath>
using namespace std;
float rt(int x){
	float num = x;
	float prm = 0;
	for (int i=0;i<8;i++){
		float ct = std::pow(10, -(i));
		while (std::pow(prm, 36) < num){
			prm += ct;
		}
		prm -= ct;
	}
	return prm;
}
int main(){
	std::cout << rt(40483056135) << std::endl;
}

File: how-to-sqrt_or_more.py
def rt(x, y = 2):
    num = x
    prm = 0 
    for i in range(16):
        ct = 10 ** -(i)
        while prm ** y < num:
            prm += ct
        prm -= ct
    return(prm)
print(rt(2))
print(rt(2, 3))
